cypress/integration/free/admin.spec.ts

```diff
@@ -2,272 +2,6 @@ import CONSTANTS from "../../support/constants";
 
 const { GOOD_PASSWORD } = CONSTANTS;
 
-const getConfig = {
```

Comment:
Moved to software.spec.ts instead of duplicated in both free/admin.spec.ts and premium/admin.spec.ts